### PR TITLE
Optimoise scenarios

### PR DESCRIPTION
**Pull request by jimjohnsonrollings**
Makes the scenario data closer to what we expect in reality

TO DO: add schema https://github.com/bbc/simorgh/issues/91

* [x] Issue linked https://github.com/bbc/simorgh/issues/51
* [x] Up-to-date with latest - `git pull --rebase origin latest`
* [x] Runs locally - `npm run dev` & [http://localhost:7080/](http://localhost:7080/)
* [x] Tests added for new features
* [x] Tests pass - `npm test`
* [x] E2E tests pass - `npm run test:e2e` (requires server running see [README](https://github.com/bbc/simorgh#tests) for details)

* [ ] Test engineer approval

### CODE REVIEW

File: data/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Simorgh Data Samples
+
+These are a set of data samples which represent possbile input to Simorgh from upstream systems (and ultimately from a CMS).  They cover a number of scenarios, most of which are made up with random Lorem Ipsum or real text.
+
+Two of the scenarios, 25 and 27, are based on real articles published on the BBC News website.
```
Comment (jamesbhobbs) [inline data/README.md]: we should have separate sub-folders for articles e.g. /data/test and /data/prod

File: data/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+Some of the scenarios are intended to represent failures - where we would reject the articles as being malformed.
```
Comment (jamesbhobbs) [inline data/README.md]: Which?

File: data/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Simorgh Data Samples
+
+These are a set of data samples which represent possbile input to Simorgh from upstream systems (and ultimately from a CMS).  They cover a number of scenarios, most of which are made up with random Lorem Ipsum or real text.
```
Comment (jamesbhobbs) [inline data/README.md]: For these data samples to be intelligible we need a schema somewhere. At very least in this PR we should mention this and mark it as a TODO: (also add a github issue). We can then update the swaggerhub.io schema we wrote a long time ago.

File: data/scenario-01.json [hunk 4]
```diff
@@ -1,5 +1,4 @@
 {
```
Comment (jamesbhobbs) [inline data/scenario-01.json:3]: We should write a test which parses these json blocks to check they're valid json

**Review by jamesbhobbs — CHANGES_REQUESTED** [review]
Few minor comments, will check linting of json after the readme is updated. The automated testing will be another ticket.

File: data/scenario-01.json (hunk 4 above)
Comment (jamesbhobbs) [reply data/scenario-01.json:3]: https://github.com/bbc/simorgh/issues/89

File: data/README.md (hunk 1 above)
Comment (jimjohnsonrollings) [reply data/README.md]: with 25 & 27 in both?

File: data/README.md (hunk 2 above)
Comment (jimjohnsonrollings) [reply data/README.md]: It wouldn't be a blind test if I told you that :P

File: data/README.md (hunk 3 above)
Comment (jimjohnsonrollings) [reply data/README.md]: https://github.com/bbc/simorgh/issues/91

**Review by jamesbhobbs — DISMISSED** [review]

File: data/README.md (hunk 3 above)
Comment (jp-bbc) [reply data/README.md]: change `possbile input` to `possible input`. Also, remove the double spacing after `CMS).`.

File: data/scenario-01.json (hunk 4 above)
Comment (jp-bbc) [reply data/scenario-01.json:3]: Agree with Benjamin

**Pull request merged**